### ISSUE TITLE
Add test: Fix misdirected animation properties.

### ISSUE THREAD

**Issue by Ms2ger** [servo/servo#6916]
@pcwalton: folloup from #6889

**Comment by pcwalton** [comment]
Yeah, I would have added a test but I don't know how to test this in a way that isn't going to be fragile, since animations progression is driven by the refresh driver.

**Comment by metajack** [comment]
Can you animate a property, wait 1s and then read the new position? You were animating the totally wrong properties before, so it seems that such a test would catch that case.

**Comment by nox** [comment]
Is this still something we need to do?
